extract-lib/src/main/java/org/icij/extract/report/HashMapReportMap.java

```diff
@@ -26,4 +26,10 @@ public boolean fastPut(final Path key, final Report value) {
 
 	@Override
 	public void close() {}
+
+	@Override
+	public boolean delete() {
+		clear();
+		return size() == 0;
```

Comment:
Can the `size` be different from 0 after the `clear` call?

Reply:
If the `clear` method fails, yes.